python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -631,6 +708,7 @@ def process_trips(db_manager: DatabaseManager) -> None:
 
     """
     update_static_version_key(db_manager)
+    update_start_times(db_manager)
```

Comment:
We should call this function after `update_static_trip_id_guess_exact`  since we are using the `VehicleTrips.trip_id` field in these queries.